### DISCUSSION TITLE
S1/2 Missing?

### DISCUSSION THREAD

**Discussion by Durahl** [EdgeTX/edgetx#1960] [Q&A] [answered]
Greetings!

I just came across an OpenTX centric YouTube Tutorial about using one of the Rotary Dials as a Global Function Volume Knob but in my attempt to recreate this feature I noticed that I'm unable of finding either the S1 or S3 Rotary Dial. The 6P one I can find no problem either via searching it or via using it while assigning a Switch but the other two are M.I.A. when browsing the Dropdown menu?

They do work on the Homescreen as turning either of them will move the two Sliders located at the bottom of the screen. They're also available as a **Source** when editing a Channel. I am though using what I assume to be [a Fork in order to enable Touch Screen functionality on my Horus X12S](https://wimalopaan.github.io/Electronics/rc/touch.html) so maybe it's a problem with that but I'm assuming it is not.

Am I missing something? 🤔

**Comment by pfeerick** [comment, marked as answer]
Possibly the order - are you setting the Switch, rather than the source? S1 and S2 should be either side of 6P. Switch should be set to ON, unless you want a switch to control whether the knob works or not. Then set Func to Volume, and then Volume to your preferred source (S1, S2, etc). And then don't forget to enable the function ;)

Using Wilhelm's fork shouldn't be an issue. You can also use a nightly for 2.8 if you wish as his PR adding this to EdgeTX was merged about three weeks ago. 

![companion_5tOpgReAGn](https://user-images.githubusercontent.com/5500713/168186359-815be12f-6d52-41c3-a798-85f538d51754.png)

![r7wNQfnY1i](https://user-images.githubusercontent.com/5500713/168186572-d9108459-bf33-47b7-9bb1-ecd52f6ac56e.png)

**Comment by Durahl** [comment]
Yeaa... I was looking for S1 under Switch 😑
Thanks for the explanation 👌
